packages/postgraphile-core/__tests__/integration/schema/__snapshots__/noDefaultMutations.test.js.snap

```diff
@@ -1798,7 +1798,7 @@ type Person implements Node {
 
     \\"\\"\\"The method to use when ordering \`PersonSecret\`.\\"\\"\\"
     orderBy: [PersonSecretsOrderBy!] = [PRIMARY_KEY_ASC]
-  ): PersonSecretsConnection! @deprecated(reason: \\"Please use personSecretByPersonId instead\\")
+  ): PersonSecretsConnection! @deprecated(reason: \\"This is deprecated.\\")
```

Comment:
- [x] Check this

Reply:
This is where we have the legacy connection for a one-to-one relation; that added a deprecation message and then this overrode it.

... 🤔 ...

Okay, yeah I think that's fine - if you deprecate the entire table then that actually wins over the connection itself being deprecated. And throwing under these circumstances does not make sense. Sold.